src/routes/admin/screenmessage/+page.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import { pb } from "$lib/stores/authStore";
+
+  export let data;
+  let screenMessages = data.screenMessages;
+  let activeMessage = data.activeMessage[0];
+
+  let open = !activeMessage.isVisible;
+  async function updateActiveMessage() {
+    await pb.collection("activeMessage").update(activeMessage.id, {
+      message: selectedMessage.id,
+      isVisible: !open
+    });
+  }
+
+  let selectedMessage;
+
+  async function updateScreenMessages() {
+    updateActiveMessage();
+    for (let message of screenMessages) {
```

Comment:
Er det noe behov for å oppdatere meldingene som ikke er aktive? Vil folk endre mer enn en av gangen?

Reply:
Man kan endre flere om gangen, jeg vet ikke om det kommer til å skje, men da blir alle oppdatert slik de skal. Alternativet krever at kun 1 input er synlig om gangen.